### ISSUE TITLE
BoF: Diversity at iEvoBio

### ISSUE THREAD

**Issue by k8hertweck** [2017-iEvoBio/organization#25]
**Discussion topic:** The iEvoBio organizers have noted a rather dramatic gender bias in attendees at our conference over the last few years, especially in regards to participants in the Software Bazaar. This BoF session will discuss possible causes and consequences of this disparity, including possible ways to minimize this bias. Further information on ways to approach this issue are outlined in this [F1000Research article about hackseq](https://f1000research.com/articles/6-197/v2).

**Comment by jhill1** [comment]
+1 for this.

**Comment by bomeara** [comment]
Agreed. I might not be able to join this (have conflicting society meeting in the am) but I'm glad you proposed this, @k8hertweck. GitHub's survey of open source, including effect of codes of conduct, negative comments, etc., might be relevant: http://opensourcesurvey.org/2017/.

**Comment by sckott** [comment]
great idea, i'm interested in this discussion

**Comment by wrightaprilm** [comment]
I'm interested in this, but going to miss pitches. Something that might be good in this discussion is: what _is_ iEvoBio. I admit to not really knowing the answer to that question. What are useful ways that someone who is not a developer on a C++ project to do some sort of complex analysis can contribute to this community? I think more clarity on this will point to more clear ways to increase diversity.

**Comment by bhaller** [comment]
Here are the notes I took during the session.

**Topic:** There's a continuing pattern in these workshops of women not presenting software, as exemplified by the iEvoBio software bazaar, which is largely (entirely?) men this year (as past years).  Why?  What do do?


**Participants:** Daisy, April, Fiona, Abigail, M. Elise, Tracy, Kathryn, Ben, Kate, Klaus.


**Take-home points:**

(1) Environmental factors are very important.  Terms like "hack-a-thon" and "software engineer" can be hard for women to identify with, driving them away.  On the other hand, a mentor, an advisor, any sort of support or encouragement can help women to feel like they belong in the software community.  Every little environmental factor makes a difference; one of the female presenters today gave her talk only because her advisor encouraged her to do so.  Approaches like project-based learning and invitations to talk could also be helpful in bringing more women and minorities in.  We should think about what we can change in our local environment and the things over which we have control.  One idea for iEvoBio is that each attendee could be asked to recruit/invite five other people they know who might belong at iEvoBio (but might not think that they do!), with an eye toward increasing diversity here.

(2) A broader appreciation for different types of work would be helpful.  Men and women often approach problems in different ways; women often try to modify existing software, collaborate with existing developers, etc., to get their work done, whereas men more often go off and develop their own software package to solve their problem un-collaboratively.  The former is often a more effective approach, but the latter approach gets more credit in terms of talks, citations, etc., ironically.  So it might be good for the former approach to receive more support and credit – for example, iEvoBio could have more talks about how existing software was used/modified/combined to create an effective pipeline, as well as the talks we have now about new software package X.


**Full notes:**

April Wright: feels comfortable fronting for the software she works on because her team is egalitarian, the whole team is interested in having it presented; seemed like it was maybe an environment that made her feel comfortable doing that.

Fiona: Spelman College, all-women's college, historically black.  She observes that there are no black people at iEvoBio; that is a diversity axis we ought to be thinking about.  She never felt there was a barrier to getting into a software, but she is in a small lab where she is the only person who can do such things.

Abigail: Works with students from impoverished backgrounds, Hispanic students, and other disadvantaged groups.  She cited a study she found about why diversity is a problem in this area: lack of peer mentors, lack of role models, lack of access to opportunities and technologies, hostile work environment, lack of computational thinking (training in thinking about problems in a computational way).  Girls are lost right at junior high.

M. Elise: Stonybrook U., moving toward computational work from organismal work.  Likes creating tools for other people.

Tracy: Data Carpentry.  Used to work in bioinformatics, has long had a mix of computation in her work.  Observes that it is hard for her to see exactly what choices she made that led her to the position she's in now.  Thinks that project-based learning might be a good way to engage under-represented groups in computation; it shows more clearly how computation work can be a way of giving back.  (A common theme among why we are doing computational work – building tools for others, making connections with others' work)

Kathryn: Colorado State, empirical biologist, prefers using other people's tools.  Helps run an undergrad mentorship program, related to diversity.  Important to focus on the proximate causes of lack of diversity at iEvoBio; we can't change the overall sexism/racism of society, but perhaps we can fix our local problem.  Game that demonstrates what women have to put up with in STEM work: phylogame.org, Women In Stem deck

April: Southeastern Louisiana U., primarily undergrad institution.  Statistical phylogenetics, very software-intensive; self-taught, which has made her realize the importance of teaching and mentoring.  

Ben: Cornell.  From a software background, which was extremely lacking in diversity.  Taking notes.  :->

Daisy: has been involved in both genetics and computer science since undergrad.  Sprinkles "computer fairy dust" across many projects, largely as a result of the connections she has made at iEvoBio.

Klaus: stats/math background.  Diversity problems are evident in many areas; r.sys.phylo is predominantly male and English-speaking, but he knows that his user base is far more diverse, so there is a mismatch there.

Kathryn: Suggestion for iEvoBio: invited speakers who are diverse, not just for the keynote, but for lightning talks, etc.  It's hard to invite people, however, since money is limited, funding people to attend is difficult, logistics are complicated, etc.

Daisy: There can be things that make women tend not to volunteer to do certain things.  "Hack-a-thon" is a term that discourages women; women may tend to say "that's not for me".  Tracy: aggressively recruiting abstracts from women can be done, but it takes time and work.  But you have to do this now to create role models for the future.  Kathryn: maybe asking each attendee to nominate five other people to attend, or give talks, or present software.  Kate: maybe if we diversified the type of talks, it would bring in more diverse speakers – not just software developer talks, but also software user talks.  Talks about how to develop a pipeline, create visualizations, etc., are useful too.  A lot of agreement that women are discouraged from giving talks because they feel that they are not "real" software developers.  Daisy: we need more scientific software developers who go out and talk to users about what tools are really needed, how the tools are used by end users, etc., because there is a disconnect between developers and users now.  And so this is another way in which talks by users at iEvoBio would be useful.  Tracy: the lack of self-identification as "developers" among women is pervasive.  Daisy: working to improve an existing package is actually often more useful than developing a new package from scratch, but women who work on existing software hesitate to identify as "developers".

M. Elise: so is there a difference in what different groups (e.g., men and women) are actually doing, or is there a difference just in how people doing the same things self-identify?  Tracy: definitely more men making R packages.  Daisy: men and women pursue different strategies to get their work done; women will work with others to make the tools do what they need to do, men will perhaps go off and write their own new package from scratch instead of collaborating, which ironically gets them more credit.

We are doing better than some fields in terms of diversity, clearly; but we still have far to go.
